### PR TITLE
docs: consume req.body in the form of key-value pairs

### PR DESCRIPTION
**Pull request by maxloh**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

The pull request add documentation on how to consume `req.body` in key-value pairs.

This may help new comers from Express.js

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by maxloh** [comment]
Hi. I found that most code snippets in readme do not contain trailing semicolon.

Should I remove semicolon from my pull request, or keep it as is?

**Comment by mcollina** [comment]
thanks, that'd be best!

File: README.md [hunk 1]
```diff
@@ -222,6 +222,9 @@ fastify.register(require('fastify-multipart'), { attachFieldsToBody: true })
 fastify.post('/upload/files', async function (req, reply) {
   const uploadValue = await req.body.upload.toBuffer()  // access files
   const fooValue = await req.body.foo.value           // other fields
+  const body = Object.fromEntries(
+    Object.keys(req.body).map((key) => [key, req.body[key].value])
+  ); // Request body in key-value pairs, like req.body in Express (Node 12+)
```
Comment (jsumners) [inline README.md:227]:
```suggestion
  ) // Request body in key-value pairs, like req.body in Express (Node 12+)
```

**Comment by maxloh** [comment]
After testing in my machine, I found that `req.body.foo.value` returns the value itself instead of a promise.

Maybe we should remove the `await` keyword?

```diff
fastify.post('/upload/files', async function (req, reply) {
  const uploadValue = await req.body.upload.toBuffer()  // access files
-  const fooValue = await req.body.foo.value           // other fields
+  const fooValue = req.body.foo.value           // other fields
  const body = Object.fromEntries(
    Object.keys(req.body).map((key) => [key, req.body[key].value])
  ) // Request body in key-value pairs, like req.body in Express (Node 12+)
})

```

**Comment by mcollina** [comment]
sure thing

**Comment by mcollina** [comment]
Can you add a unit test for this? So it's sure we won't regress.

**Comment by maxloh** [comment]
> Can you add a unit test for this? So it's sure we won't regress.

Is the following line enough? Should I write another test for that?

https://github.com/fastify/fastify-multipart/blob/d3790a19d5dbfc042aaa6edfe468664007042972/test/multipart-body-schema.test.js#L45

**Comment by mcollina** [comment]
actually it is! Landed

**Pull request merged**